### DISCUSSION TITLE
Clarification regarding the performance of dynamic mode

### DISCUSSION THREAD

**Discussion by nickpainter** [darkreader/darkreader#9461] [Q&A] [answered]
The UI states that Filter+ uses hardware acceleration/GPU to ensure performance.  Does dynamic mode primarily use GPU or CPU resources?

**Comment by Gusted** [comment, marked as answer]
Dynamic mode uses CPU. The way that Filter(+) can utilize the GPU is purely due to the browser optimizing "image" operations(aka filters, hence the name) via the GPU.
